eu_dgc_v1_schema.yaml

```diff
@@ -97,20 +98,10 @@ properties:
               example: 121212-1212
       dob:
         title: Date of birth
-        description: Mandatory if no Person identifier is provided
+        description: The date of birth for of the person addressed in the certificate
```

Comment:
minor: "The date of birth for of the person" -> "The date of birth of the person"